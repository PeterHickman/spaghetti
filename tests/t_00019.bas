10 REM Testing gosub
20 print 1
30 gosub 100
40 print 2
50 gosub 100
60 print 3
70 gosub 100
80 print "Ok"
90 end
100 print "In gosub"
110 return
