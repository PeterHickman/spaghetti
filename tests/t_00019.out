1
In gosub
2
In gosub
3
In gosub
Ok
